x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/cache/full/SearchableSnapshotsPrewarmingIntegTests.java

```diff
@@ -279,7 +281,7 @@ public void onFailure(Exception e) {
 
         logger.debug("--> waiting for background cache to complete");
         assertBusy(() -> {
-            if (threadPool.executor(SearchableSnapshots.CACHE_FETCH_ASYNC_THREAD_POOL_NAME) instanceof ThreadPoolExecutor executor) {
+            if (threadPool.executor(SearchableSnapshots.CACHE_FETCH_ASYNC_THREAD_POOL_NAME)instanceof ThreadPoolExecutor executor) {
```

Comment:
I think you are using an old Spotless version, see https://github.com/elastic/elasticsearch/issues/82794.

```suggestion
            if (threadPool.executor(SearchableSnapshots.CACHE_FETCH_ASYNC_THREAD_POOL_NAME) instanceof ThreadPoolExecutor executor) {
```